backend/src/server/routes/v3/secret-router.ts

```diff
@@ -166,6 +166,11 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         workspaceSlug: z.string().trim().optional().describe(RAW_SECRETS.LIST.workspaceSlug),
         environment: z.string().trim().optional().describe(RAW_SECRETS.LIST.environment),
         secretPath: z.string().trim().default("/").transform(removeTrailingSlash).describe(RAW_SECRETS.LIST.secretPath),
+        expand: z
+          .enum(["true", "false"])
+          .default("false")
+          .transform((value) => value === "true")
+          .describe(RAW_SECRETS.LIST.expand),
```

Comment:
intentionally made this false so that we can control the migration towards this pattern of use. my main concern is in how this would affect overall server latency and resource utilization. I'm thinking for this one we can start slow via the Github action I'm currently working on